### PR TITLE
PrEP branch

### PR DESCRIPTION
**Pull request by Nfratt**
Ricardo and I paired, we where able to get the  Functionality done. The next step is to talk to design to get one more congruent with the rest of the site

### CODE REVIEW

File: src/components/PrEP.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <p> Since PrEP is just for HIV prevention, you still need to take precautions like barrier methods and regular testing to prevent other STIs/STDs.</p>
+                <p> Side effects like nausea, loss of appetite, and headaches may occur, but usually go away over time</p>
+                <p>Once you’re on PrEP, you’ll need to visit your doctor once every three months for follow-ups on how the medication is working and an HIV test. 
+                    This may take the form of an in-person visit or a home test and telemedicine visit.</p>
+                <p> If you’ve been exposed to HIV and aren’t on PrEP, you can take PEP (post-exposure prophylaxis) within 72 hours of exposure. The sooner you can get
+                     PEP, the more effective it is, and it must be taken daily for 28 days.</p>
+            </Accordion>
+            <Accordion title="Sources">
+            <div className="sources-links">
+                <p>CDC. (2020, November 3). About Prep. HIV Basics.<a href="https://www.cdc.gov/hiv/basics/prep/about-prep.html" target="_blank" without rel="noopener noreferrer">
```
Comment (alcastonguay) [inline src/components/PrEP.jsx:43]: The "without" in the <a> tag is giving an error in the console when the app is run - 
```Warning: Received `true` for a non-boolean attribute `without`.
If you want to write it to the DOM, pass a string instead: without="true" or without={value.toString()}.```

**Comment by alcastonguay** [comment]
I was just curious if you intended to merge this with main? It sounds like it is not necessarily complete?

**Comment by Nfratt** [comment]
Correct! it is not ready to be merged yet, I just wanted to get it up to git hub and saved before moving forward on it as it needed to sit until a meeting with design.

File: src/components/PrEP.jsx (hunk 1 above)
Comment (Nfratt) [reply src/components/PrEP.jsx]: Thanks I removed the with outs the didn't belong there.

**Comment by Nfratt** [comment]
Need some help reviewing this one there are a lot of conflicting changes.

**Comment by alcastonguay** [comment]
If you merge development into PrEP, you will be able to handle those conflicts on the PrEP branch instead of the development. Let me know if you have any questions about this!

**Review by alcastonguay — APPROVED** [review]
Reviewed conflicts with Nick, good to go!

**Pull request merged**